testing/scenario_app/runtime_mode.gni

```diff
@@ -0,0 +1,7 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/common/config.gni")
+
+is_aot = flutter_runtime_mode == "profile" || flutter_runtime_mode == "release"
```

Comment:
Is it possible to assert at the top-level of this file that the target OS is iOS or Android?

Reply:
Done